CMakeLists.txt

```diff
@@ -39,36 +39,36 @@ find_package(Threads REQUIRED)
 find_package(CURL REQUIRED)
 find_package(jsoncpp)
 
-add_library(restclient-cpp SHARED
-  source/restclient.cc
-  source/connection.cc
+add_library(restclient-cpp SHARED
+  source/restclient.cc
```

Comment:
I think my editor just automatically made these spacing changes, I am happy to change them back if you'd like.